resources/views/hardware/bulk-checkin.blade.php

```diff
@@ -0,0 +1,115 @@
+@extends('layouts/default')
+
+ Page title
```

Comment:
We should remove this line right?

![CleanShot 2023-09-20 at 11 40 38@2x](https://github.com/snipe/snipe-it/assets/1141514/9aa985ba-324b-40dc-9899-30541901c6eb)